Ghidra/Processors/PowerPC/data/languages/Scalar_SPFP.sinc

```diff
@@ -470,3 +467,97 @@ define pcodeop ConvertFloatingPointFromUnsignedFraction;
 {
   CRFD = A:4 f< B:4;
 }
+
```

Comment:
I think these are probably better moved to SPE_EFV.sinc

Reply:
I should clarify. This will not build as is because the vector instructions rely on the registers being greater than 4 bytes in length. Our 32-bit PPC processors will fail to build with this included here. Moved to SPE_EFV.sinc will resolve the issue.